networks/typical/main.tf

```diff
@@ -13,51 +13,69 @@ provider "docker" {
 
 locals {
   number_of_nodes = var.number_of_nodes
-  node_indices    = range(local.number_of_nodes)
+  node_indices = range(local.number_of_nodes)
   more_args = join(" ", [
-    "--allow-insecure-unlock" # since 1.9.7 upgrade
+    "--allow-insecure-unlock",
+    # since 1.9.7 upgrade,
```

Comment:
```suggestion
    "--allow-insecure-unlock", # since 1.9.7 upgrade
```
the comment feels not related to the line anymore, is it possible to put like this?

Reply:
I will fix in a separate branch (used auto merge so as you approved, it merged).

Reply:
fixed on https://github.com/ConsenSys/quorum-acceptance-tests/pull/121